airflow/providers/databricks/hooks/databricks.py

```diff
@@ -152,29 +152,38 @@ def submit_run(self, json: dict) -> int:
         response = self._do_api_call(SUBMIT_RUN_ENDPOINT, json)
         return response["run_id"]
 
-    def list_jobs(self, limit: int = 25, offset: int = 0, expand_tasks: bool = False) -> list[dict[str, Any]]:
+    def list_jobs(
+        self, limit: int = 25, offset: int = 0, expand_tasks: bool = False, job_name: str | None = None
+    ) -> list[dict[str, Any]]:
         """
         Lists the jobs in the Databricks Job Service.
 
         :param limit: The limit/batch size used to retrieve jobs.
         :param offset: The offset of the first job to return, relative to the most recently created job.
         :param expand_tasks: Whether to include task and cluster details in the response.
+        :param job_name: Optional name of a job to search.
         :return: A list of jobs.
         """
         has_more = True
         jobs = []
 
         while has_more:
-            json = {
+            jsn = {
```

Comment:
How about we call this `json_data`? I feel it's better than an abbreviation. No strong opinion here